powdr_cli/src/main.rs

```diff
@@ -599,7 +599,27 @@ fn handle_riscv_asm<F: FieldElement>(
             );
         }
         (false, true) => {
-            unimplemented!("Running witgen with continuations is not supported yet.")
+            log::info!("Dry running execution to collect bootloader inputs...");
+            let bootloader_inputs = rust_continuations_dry_run(file_name, contents, inputs.clone());
+            let num_chunks = bootloader_inputs.len();
+
+            bootloader_inputs
+                .into_iter()
+                .enumerate()
+                .for_each(|(i, bootloader_inputs)| {
+                    log::info!("Running chunk {} / {}...", i + 1, num_chunks);
+                    let pipeline = Pipeline::default()
+                        .with_output(output_dir.to_path_buf(), force_overwrite)
+                        .from_asm_string(contents.to_string(), Some(PathBuf::from(file_name)))
+                        .with_prover_inputs(inputs.clone())
+                        .with_backend(BackendType::PilStarkCli);
```

Comment:
shouldn't this use `prove_with`?

Reply:
True. It get's set again later, so I just removed this line.